### DISCUSSION TITLE
How can I send to my own endpoint the tracking data?

### DISCUSSION THREAD

**Discussion by luisgmfarias** [DavidWells/analytics#424] [Q&A] [answered]
I don´t know if I am in the right path, but I would like to create a plugin to send tracking data to my private endpoint. Can someone give me some advice?

**Comment by DavidWells** [comment, marked as answer]
See https://github.com/elvisace/analytics-fetch/blob/master/index.ts or checkout the docs on creating a plug-in here https://getanalytics.io/plugins/writing-plugins/
